aspnet/migration/rc1-to-rc2.rst

```diff
@@ -172,24 +186,34 @@ The ASP.NET Core Module must be configured in ``web.config``:
 The Publish IIS tool 
 ^^^^^^^^^^^^^^^^^^^^^
 
-The name of the package that contains the Publish IIS tool was changed to ``Microsoft.AspNetCore.Server.IISIntegration.Tools``. This requires changing your project.json file to inlude the ``Microsoft.AspNetCore.Server.IISIntegration.Tools`` package instead of the ``dotnet-publish-iis`` package.
+The name of the package that contains the Publish IIS tool was changed to ``Microsoft.AspNetCore.Server.IISIntegration.Tools``. This requires changing your ``project.json`` file to inlude the ``Microsoft.AspNetCore.Server.IISIntegration.Tools`` package instead of the ``dotnet-publish-iis`` package.
 
 You can use the Publish IIS tool to publish your app with the ``web.config`` file that is required for your target environment. More information about Publish IIS changes is at https://github.com/aspnet/Announcements/issues/164.
 
-IIS integration middleware is now setup using ``WebHostBuilder`` in ``Program.Main``, and is no longer called in the ``Configure`` method of the ``Startup`` class. 
+IIS integration middleware is now setup using ``WebHostBuilder`` in ``Program.Main()``, and is no longer called in the ``Configure()`` method of the ``Startup`` class. 
 
 .. code-block:: c#
 
   public static void Main(string[] args)
   {
-    var host = new WebHostBuilder().UseIISIntegration();
+    var host = new WebHostBuilder().UseIISIntegration().Build();
   }
+  
 
+Web Deploy changes
+^^^^^^^^^^^^^^^^^^^^^^^  
+Delete ``RC1StarterWeb - Web Deploy-publish.ps1``. This is a script generated by VS for web deploy. There is a version for RC1 projects (dnx based) and a different script for RC2 projects (dotnet based) which are incompatible with each other. As such, when migrating to RC2, you need to delete the old script and let VS generate a new one to ensure web deploy works for the converted RC2 project.
```

Comment:
Some sections have an empty line between the heading and the first line of text, does this matter?